content/PGP/Card_edit.adoc

```diff
@@ -12,11 +12,18 @@ To configure the OpenPGP application on a YubiKey you can use GnuPG.
 
 == Prerequisites
 
-Make sure you have a recent YubiKey with firmware version 3.1.8 or
-later; use "lsusb -v" to find out.
+Installed the link:https://developers.yubico.com/yubikey-personalization/Manuals/ykpersonalize.1.html[Yubikey personalization tool].
 
-Make sure the device is in OTP/CCID or CCID mode, use the YubiKey
-personalization tool "ykpersonalize -m82" to switch.
+Verify that your YubiKey has a firmware version 3.1.8 or later.
+
+  $ ykpersonalize
```

Comment:
Ah, good point.

I'll revert back to lsbusb (better with `lsusb -v -d 1050:` and mention that 1050 is the Yubico USB vendor id I think).